apps/portals/adknowledgeportal/vite.config.ts

```diff
@@ -0,0 +1,25 @@
+import { portalsViteConfig } from 'vite-config'
+import { mergeConfig } from 'vite'
+import { resolve } from 'path'
+
+const projectRootDir = resolve(__dirname)
+
+export default mergeConfig(portalsViteConfig, {
```

Comment:
The Vite configuration for each portal is identical

Reply:
So after this change, it looks like there will be at least a few files that are duplicated in each project (index.html and vite.config.ts).  It looks like you've done your best to reuse as much as possible (via templating).  You clearly share my concern for future drift (where a fix is made in a subset of projects).  What are other ways to mitigate this issue?  Add a loud comment to each instance indicating that the file should not be modified, unless all instances are modified?

Reply:
[Done!](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/925/commits/1660e0b84bce223485075f5bed5e65c79f89c974)